### ISSUE TITLE
Please remove the script.video.F4mProxy-2.2.6.zip and plugin.video.indianott.zip from main repo

### ISSUE THREAD

**Issue by siuside** [gagandeep/indian_tv_ott#1]
They should be under "Releases". That way when you update the code of indian_tv_ott repo, we can download new "master" zip/cone of the entire repo without these zip files. 

Thanks for the addon! I have yet to try it but will do soon.

**Comment by gagandeep** [comment]
Agree, I will take care of this in next release. The target for next release another version on this weekend.

**Comment by siuside** [comment]
Great, thanks. Also since this doesn't seem to be playable even with a .in IP, can you please tell if this actually worked for you without any proxies, and if so which geo area?

Starsports would be a great addition. They have closed subscription model and now work via just geo blocking I believe which individuals can overcome based on their VPNs or other arrangements.

**Comment by gagandeep** [comment]
Its working for me even right now. Do you installed F4MProxy addon?

**Comment by siuside** [comment]
Yes I have the latest from Shani's repo. You didn't mention "where" does it work from even right now.

**Comment by gagandeep** [comment]
Mumbai, India

**Comment by siuside** [comment]
Ok I will try with a Mumbai VPN to test

**Comment by siuside** [comment]
Ok worked fine after connecting to .IN VPN. I had to modify some of my dns settings which is why it may not have worked last time. I believe I can get it to work just with DNS services, but it requires quite a few domains to be mapped to India.

**Comment by siuside** [comment]
This is the error when you try to play when not in .in GEOIP range

http://pastebin.com/wAJVcSt5

**Comment by gagandeep** [comment]
Thanks, this will help me a lot.
